src/DigitalProductPassport/FoodArtifact/DataSheet_v0.1.py

```diff
@@ -0,0 +1,95 @@
+from definition_tooling.converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class FatContentType(CamelCaseModel):
+    fats: float = Field(
+        ...,
+        title="Fats",
+        description="The amount of fat per 100g measured in grams",
+        example=6,
+    )
+    saturated_fats: float = Field(
+        ...,
+        title="Saturated Fats",
+        description="The amount of saturated fat per 100g measured in grams",
+        example=0.4,
+    )
+
+
+class EnergyContentType(CamelCaseModel):
+    energy: int = Field(
+        ...,
+        title="Energy",
+        description="The amount of energy per 100g measured in kilojoules",
+        example=750,
+    )
+    calories: int = Field(
+        ...,
+        title="Calories",
+        description="The amount of calories per 100g measured in kilocalories",
+        example=180,
+    )
+
+
+class FoodArtifactDataSheetRequest(CamelCaseModel):
+    product: str = Field(
+        ...,
+        title="Product identifier",
+        description="Technical product identifier used by the manufacturer",
+        example="battery-100wh-s",
+    )
+    id: str = Field(
+        ...,
+        title="Identifier",
+        description="Unique identifier for the product",
+        example="177389-09633",
+    )
```

Comment:
The sheet was slightly updated, this should conform with the current content in it:

```suggestion
    product: str = Field(
        ...,
        title="Product code",
        description="The product code used for identifying the product type",
        example="french-fries-500g",
    )
    id: str = Field(
        ...,
        title="Identifier",
        description="Unique identifier of the product",
        example="550e8400-e29b-41d4-a716-446655440000",
    )
```